packages/hub/node-tests/services/scheduled-payments/validator-test.ts

```diff
@@ -1,10 +1,53 @@
 import { ScheduledPayment } from '@prisma/client';
 import { Decimal } from '@prisma/client/runtime';
-import { setupHub } from '../../helpers/server';
+import { registry, setupHub } from '../../helpers/server';
+import { Networkish } from '@cardstack/cardpay-sdk';
+
+class StubCardpaySDK {
+  getConstantByNetwork(name: string, network: Networkish) {
+    if (!network) {
+      throw new Error(`network can't be null`); 
+    }
+
+    switch(name) {
+      case 'scheduledPaymentFeeFixedUSD':
+        return 0.25;
+      case 'scheduledPaymentFeePercentage':
+        return 0.1;
+      default:
+        throw new Error(`unsupported mock cardpay`);
+    }
+  }
+
+  fetchSupportedGasTokens(network: Networkish) {
+    if (!network) {
+      throw new Error(`network can't be null`); 
+    }
+
+    return [
```

Comment:
The shape of this return value is actually just what is in the `TokenDetail` interface. The example here is the data that the relay server returns but the implementation of fetchSupportedGasTokens returns a subset of it.

Reply:
updated